src/server/environment.js

```diff
@@ -2,6 +2,8 @@ const { join, resolve } = require('path')
 
 const VERBOSE = Boolean(process.env.VERBOSE)
 
+const DB_URL = process.env.POSTGRES_URL
```

Comment:
Could we just make this `POSTGRES_URL`?  I don't see any reason for there to be two different names that developers have to remember match up.